airavata-services/profile-service/profile-service-server/src/main/java/org/apache/airavata/service/profile/handlers/IamAdminServicesHandler.java

```diff
@@ -272,8 +274,8 @@ private PasswordCredential getTenantAdminPasswordCredential(String tenantId) thr
             csClientPool.returnResource(csClient);
             registryClientPool.returnResource(regClient);
             return TenantAdminPassword;
-        }catch (Exception e){
-            logger.error("Error while getting the tenant admin password", e);
+        } catch (Exception e){
+            logger.error("Error while getting the tenant admin password for the tenant id "+tenantId+"", e);
```

Comment:
Keep spaces before and after +. id " + tenantId. Remove trailing "" as it is useless